### PR TITLE
Update compliance history table

### PR DESCRIPTION
**Pull request by clyang82**
1. update table to add cluster_id column
2. add `compliance_changed_frequency` to indicate how frequent the compliance status is changed

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/database/4.trigger.sql [hunk 1]
```diff
@@ -67,3 +67,8 @@ DROP TRIGGER IF EXISTS set_timestamp ON spec.policies;
 CREATE TRIGGER set_timestamp BEFORE UPDATE ON spec.policies FOR EACH ROW EXECUTE FUNCTION public.trigger_set_timestamp();
 DROP TRIGGER IF EXISTS set_timestamp ON spec.subscriptions;
 CREATE TRIGGER set_timestamp BEFORE UPDATE ON spec.subscriptions FOR EACH ROW EXECUTE FUNCTION public.trigger_set_timestamp();
+
+DROP TRIGGER IF EXISTS update_compliance_table ON local_status.compliance;
+CREATE TRIGGER update_compliance_table AFTER INSERT ON local_status.compliance FOR EACH ROW EXECUTE FUNCTION public.set_cluster_id_to_compliance();
+DROP TRIGGER IF EXISTS update_compliance_table ON status.compliance;
+CREATE TRIGGER update_compliance_table AFTER INSERT ON status.compliance FOR EACH ROW EXECUTE FUNCTION public.set_cluster_id_to_compliance();
```
[the remark this answers is not in the document]
Comment (yanmxa) [reply operator/pkg/controllers/hubofhubs/database/4.trigger.sql:74]: Maybe we can remove the `set_cluster_id_to_compliance` trigger. Cause we have specified the `cluster_id` in the code.

[the remark this answers is not in the document]
Comment (clyang82) [reply operator/pkg/controllers/hubofhubs/database/4.trigger.sql:74]: the cluster_id we handled is in status.managed_clusters. for the policy, there is no cluster_id information. the status only contains the cluster_name. that is why using the triggers to help on this case.

**Pull request merged**